server/src/models/constants.ts

```diff
@@ -19,6 +19,7 @@ export enum UserRegExps {
     CountriesData = '/countries',
     AvailableCountries = '/available',
     CountryData = '/country',
+    Trends = '/trends',
```

Comment:
You, probably, forgot to add description of this command to the **helpResponse.ts** where we give brief overview of all available commands for the user